blocks/core/ff_module/ff_module-form-input/ff_module-form-input.js

```diff
@@ -2,10 +2,23 @@
 
 var React = require('react');
 
+function generateClass(base, props) {
+    var classNames = [];
+    props = props || {};
+    classNames.push(base);
+    if (!!props.modifier) classNames.push(base + '--' + props.modifier);
+    if (!!props.classes) classNames.push(props.classes);
+    if (!!props.className) classNames.push(props.className);
+    return classNames.join(' ');
+}
+
 module.exports = React.createClass({
 	displayName: 'FormInput',
 	render: function() {
-		var attributes = {};
+
+		var attributes = {},
+            className = generateClass('ff_module-form-input', this.props);
```

Comment:
Tiny: This might just be github's diff view, but the indentation is weird on this line. I think it might be spaces where everything else is tabs.

It might be worth using something like http://editorconfig.org/ to keep this consistent even across all the various editors that people use, but that's beyond the scope of this PR :P